src/static/css/module/loan-comparison.less

```diff
@@ -364,63 +364,138 @@
   --------------------------- */
 
 .comparison-results {
-  margin-top: 60px;
+  .expandable {
+    .respond-to-min(@tablet-min, {
+      overflow: hidden;
+    });
+  }
+  .expandable_header {
+    margin-bottom: @grid_gutter-width/2;
+    background: @gray-10;
+    .respond-to-min(1000px, {
+      margin-bottom: @grid_gutter-width;
+    });
+  }
+  .expandable_link {
+    position: relative;
+    z-index: 1;
+  }
+  .expandable__padded,
+  .expandable__padded.expandable__expanded {
+    background: @white;
+  }
+  .expandable__padded .expandable_content {
+    margin-right: 0;
+    margin-left: 0;
+    background: @white;
+    &:before,
+    &:hover {
+      display: none;      
+    }
+  }
 }
 
 .result-heading {
-  color: @gray;
   .grid_push(2);
 }
 
 .results-section {
+  position: relative;
   margin-bottom: 60px;
 }
-
-.results-section-heading {
-  .reset;
-  .webfont-medium();
-  font-size: 1.125em;
-  background-color: @gray-10;
-}
-
-.lc-primary-result {
-  font-size: 1.375em;
-  .webfont-regular();
-}
-
-@media (min-width: 600px) {
-  .results-section {
-    background-color: @gray-10;
-    padding: 12px 0;
+  .results-section-heading {
+    margin: 0;
+    .cf-icon {
+      color: @navy;
+    }
+    .respond-to-min(@tablet-min, {
+      font-size: 1.375em;
+    });
+  }
+
+  .primary-results-row {
+    margin-bottom: @grid_gutter-width;
+    .respond-to-min(1000px, {
+      position: absolute;
+      top: 8px;
+    });
+  }
+
+  .detail-results-row {
+    padding: 0 0 25px; // magic numberssssss
+    .respond-to-min(@tablet-min, {
+      padding-left: @grid_gutter-width/2;
+    });
+    .h6 {
+      color: @darkgray;
+    }
+    h6 {
+      padding: 0 @grid_gutter-width/2;
+      .respond-to-min(@tablet-min, {
+        .meta-header;
+        padding-right: 0;
+        padding-left: 0;
+        margin-right: @grid_gutter-width/2;
+        margin-left: @grid_gutter-width/2;
+        color: @darkgray;
+      });
+    }
   }
 
-  .results-section-heading,
-  .lc-result {
-    .grid_column(3);
+  .detail-results-row__nested {
+    .respond-to-min(@tablet-min, {
+      padding-left: @grid_gutter-width;
+    });
   }
 
-  .results-section-heading {
-    font-size: 1.375em;
+  .primary-results-row,
+  .detail-results-row {
+    width: 100%;
+    .respond-to-min(@tablet-min, {
+      margin-left: 25%;
+    });
```

Comment:
I tried doing this but then the grid doesn't line up with the 1/4 columns in the loan form above.

I'll keep it as is for now and try again when we merge and see if I can get these elements using the same grid.